tools/db_bench_tool.cc

```diff
@@ -1447,13 +1566,9 @@ class RandomGenerator {
     return Slice(data_.data() + pos_ - len, len);
   }
 
-  Slice GenerateWithTTL(unsigned int len) {
```

Comment:
Why this function is removed?

Reply:
1. GenerateWithTTL is dead code, no one calls it.
2. GenerateWithTTL does the same with thing as 'Slice Generate(unsigned int len)'